### PR TITLE
Build/test cleanups

### PR DESCRIPTION
**Pull request by tdsmith**
Clean up some build and test config.

Importantly, this makes docs warnings errors again, so this should fail until we patch that up.

### CODE REVIEW

**Comment by felixlawrence** [comment]
I don't understand this area of packaging well enough to know if we're losing something useful; it would be more useful to get an engineer to review it?

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ pip install mozanalysis
 
 ## Testing locally
 
+### with Tox
+
+Install tox into your global Python environment and run `tox`.
```
Comment (felixlawrence) [inline README.md]: Might be friendly to point out that while they're hacking, people can run a single file of tests by something like:
```
tox -e py37 tests/stats/test_bootstrap.py
```

**Comment by tdsmith** [comment]
> I don't understand this area of packaging well enough to know if we're losing something useful; it would be more useful to get an engineer to review it?

If it hasn't broken your workflow, we haven't. :) They're in git history if we decide we need them.

File: README.md (hunk 1 above)
Comment (tdsmith) [reply README.md]: Added a couple examples.

**Pull request merged**